content/reference/drops/time_drop.md

```diff
@@ -0,0 +1,14 @@
+# Time Drop
+
+Return attributes with time. You have ability to manage time format.
```

Comment:
Ok we name it after `Time.zone`
@hash4di  It has to stay then, but it should only describe `today` and `tomorrow`